autogtp/README.md

```diff
@@ -30,10 +30,42 @@ directly.
 
 # Running
 
-Copy the compiled leelaz binary into the autogtp directory, and run
-autogtp.
+autogtp and leelaz binaries need to be in the same directory.
+
+While autogtp is running, typing q+Enter will save the processed data and exit. 
+When autogtp runs next, autogtp will continue the game.
+
+# Help 
+
+For more details about AutoGTP, or for specific use (for example how to use multi GPU, 
+or any other setting), you can run, while in the directory where your autogtp executable is :
+- `./autogtp --help` on linux/mac
+- `autogtp.exe --help` on windows
+
+This will display a list of all possible settings and how to use them
+
+For your convenience, and for easier reference, a copy is provided below :
```

Comment:
Please don't do this, or it has to be updated each time one of the command line options changes.

Reply:
this would add maintainance indeed, but i thought it would serve some purpose

ok then, will remove this :+1: 
we may link to specific github issues everytime it is needed rather then